config/config_default.yml

```diff
@@ -11,54 +12,12 @@ core:
             - iridium
             - telemetry
     config_save_interval: 60
-    dump_interval: 10
```

Comment:
keep `dump_interval` under the `core` key b/c it is the interval on which `core` tells `telemetry` its time to send info down in a dump